nilearn/datasets/description/main.rst

```diff
@@ -0,0 +1,37 @@
+Montreal Artificial Intelligence and Neuroscience conference 2018 datasets
```

Comment:
I think that I'd like the line above to be change to match better the new name, as well as the name of the file.

I general, I'd like us to avoid using "main" in the code and the filenames to describe this dataset, I will be confusing to people.